evap/contributor/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.conf.urls import url
 
-from evap.contributor.views import *
+from evap.contributor.views import index, profile_edit, course_view, course_edit, course_preview
```

Comment:
i would argue that this is a valid import of everything (because every view will get a url) and might even be better. but okay, let's do it for landscape ;)